_docs/concepts/security/authn-policy.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Istio authentication policy
+overview: Describes Istio Authentication policy
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+## Overview
+Istio authentication policy enables admin to specify authentication requirements for a service (or services). Istio authentication policy is composed of two-parts authentication:
+
+* Peer: verifies the party, the direct client, that makes the connection. The common authentication mechanism for this is [mutual TLS]({{home}}/docs/concepts/security/mutual-tls.html). Istio will be responsible for managing both client and server sides to enforce the policy.
+
+* Origin: verifies the party, the original client, that makes the request (e.g end-users, devices etc). JWT is the only supported mechanism for origin authentication at the moment. Istio will configure the server side to perform authentication, but will not enforce the client side to send the required token.
```

Comment:
will configure -> configures
will not enforce the client side to send -> doesn't enforce that the client side sends